### ISSUE TITLE
Number of blocked impulses

### ISSUE THREAD

**Issue by konstantin-morenko** [raicem/impulse-blocker#27]
It would be interesting to know the number of impulses, blocked by this extension for each site and overall.

**Comment by Crote** [comment]
To add to this, perhaps keep track _when_ they were blocked. It would be nice if we could graph the impulse count on, say, a day-by-day base.

**Comment by neuhaus** [comment]
Yes it would also be great to see for how many minutes we paused the extension each day.
